tensorflow/python/keras/utils/data_utils.py

```diff
@@ -221,6 +223,9 @@ def get_file(fname,
   Returns:
       Path to the downloaded file
   """
+  if origin is None:
+    raise ValueError("Please specify the 'origin' argument (URL of the file "
```

Comment:
Nit: use `'` as quote character for consistency with the rest of the file

Reply:
@fchollet do you mean writing:
`'Please specify the \'origin\' argument (URL of the file '`
instead of
`"Please specify the 'origin' argument (URL of the file "`?

Reply:
Or just use `"origin"`. But yes.

Reply:
@fchollet, I will do it. However I would just like to point out that of all the error messages present in the tensorflow project, 3321 use double quotes, while only 1603 use single quotes.